docs/cody/clients/cody-with-sourcegraph.mdx

```diff
@@ -2,12 +2,12 @@
 
 <p className="subtitle">Learn how to use Cody in the web interface with your Sourcegraph.com instance.</p>
 
-<Callout type="info"> The new chat UI for Cody for web is currently in Beta and is available to users on Sourcegraph versions >=5.5.</Callout>
+<Callout type="info"> The new unified chat UI for Cody for web is currently available to users on Sourcegraph versions >=5.5.</Callout>
```

Comment:
- Users must be on JB version 2023.2 or above in order to get these changes (and future updates)! 

- We’ll continue to support Cody for JB versions below 2023.2 but only with critical bug fixes. We won’t be adding new features or improvements to those older versions.

Reply:
perhaps we should remove the word unified, we call it unified internally but to our customers it's just a new UI. Even "new UI" is a bit misleading, maybe refreshed Cody Web UI?

Reply:
In that case, let's not call it anything. The messaging should show that our chat experience is top-notch by default. Instead, I can add a callout that you need to be on the latest JB version for this new chat UI.